### DISCUSSION TITLE
Additional File Extensions

### DISCUSSION THREAD

**Discussion by ambidustrious** [solspace/craft-freeform#183] [Questions & Answers] [answered]
I'm sure this is a silly question, but I need to add an extension to the allowed file types for the file upload field. I've looked everywhere, but I don't see a way to do this. Looking to add "csv" as a file type.  

Any insight appreciated.

Thanks

**Comment by kjmartens** [comment, marked as answer]
Try allowing `Text` on the file upload field in the form builder. I think `.csv` will fall under that category. :)
